hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -27,7 +27,6 @@
   sim_tops: ["clkmgr_bind",
              "pwrmgr_bind",
              "rstmgr_bind",
-             "pwrmgr_rstmgr_bind",
```

Comment:
The better pwrmgr_rstmgr_sva_if is bound in top_earlgrey_bind below, and it is bound to top_earlgrey so it can connect all ports.